rules/no-hex-escape.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+const fix = value => {
```

Comment:
Not sure if @sindresorhus prefers to remove this empty line. If he prefers that, maybe we should write a rule for that :D.